src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const expenseService = require('../services/expenses.service');
+
+const get = (req, res) => {
+  res.send(expenseService.getExpenses());
+};
```

Comment:
this request can contain the parameters
<img width="532" alt="image" src="https://github.com/mate-academy/node_accounting-app-v2/assets/98764641/6c04e128-46ec-43df-8541-febf4321ad2d">

Reply:
not fixed